src/components/pip/pip-parent.tsx

```diff
@@ -46,11 +46,6 @@ export class PipParent extends Component<PIPParentComponentProps> {
       }
     }
 
-    const videoContainerStyles = {
-      height: `${props.playerHeight + 'px'}`,
-      width: `${props.playerWidth + 'px'}`
-    };
-
-    return <div className={styleClass.join(' ')} style={videoContainerStyles} ref={this.videoContainerRef} />;
+    return <div className={styleClass.join(' ')}  ref={this.videoContainerRef} />;
```

Comment:
remove also `playerHeight` and `playerWidth` from mapStateToProps and component props.

Reply:
done

Reply:
@JonathanTGold the PR approved, please be sure you merged changes from latest master into current branch before merge the PR. Lian made some changes related to video-container size, so we don't want get any regression for the component.

Reply:
@lianbenjamin please, review the PR.

Reply:
approved - when i merged the changes to dualscreen related to ratio of the pip player, i tested it with current PR changes, to make sure they do not collide. @JonathanTGold please double check and make sure both functionalities work as expected against latest code.